sdk/apimanagement/api-management-custom-widgets-tools/review/api-management-custom-widgets-tools.api.md

```diff
@@ -75,6 +83,16 @@ export type Secrets = {
     apiVersion: string;
     userId?: string;
     token?: string;
+    parentLocation: {
```

Comment:
this seems pretty cumbersome for an interface - I'm guessing it's the public shape of `URL`, but in that case why not require a `URL` directly?

Usually though, we allow customers to pass a plain string for urls

Reply:
Not sure, if I understand what you mean.
This is pretty much a stringified location object - `JSON.parse(JSON.stringify(window.location))`
This is only a data structure we are sending "down" to the customer, I don't expect this change to break anything, as they are not (meant to) create those objects themself.

Reply:
wouldn't `window.location.href` contain all the information you need? The origin/protocol/etc are all parsed out of the href if I'm not mistaken.

Reply:
or `window.location.toString()` but I think that is the same.

Reply:
yes, that's true, but `location` is technically a standardized object so I wanted to give our customers convenience of having a familiar option to use, for example `parentLocation.hostname` etc.

Reply:
Since this is a type you are returning rather than something the user has to worry about assembling, I'm okay with it having extra/redundant information.